lib/foo.go

```diff
@@ -0,0 +1,5 @@
+package lib
+
+func F(x int) int {
```

Comment:
__[staging][gometalinter]__
* [golint] exported function F should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->